includes/modules/pages/checkout_confirmation/header_php.php

```diff
@@ -52,7 +52,9 @@
 
 
 if (DISPLAY_CONDITIONS_ON_CHECKOUT == 'true') {
-    $_SESSION['conditions'] = $_POST['conditions'] ?? NULL;
```

Comment:
What happened to setting that into the session?

Reply:
It was not in the original
```
     if (DISPLAY_CONDITIONS_ON_CHECKOUT=='true') {
 -     if (!isset ($_POST['conditions'])||($_POST['conditions']!='1')) {
 -       $messageStack->add_session ('checkout_payment', ERROR_CONDITIONS_NOT_ACCEPTED, 'error');
 -     }
 - 
 +        $_SESSION['conditions'] = $_POST['conditions'] ?? NULL;
```

 Refactor T&C #5019

Happy to put it in if you think it is required.

Reply:
Thinking a bit more about it, I do not think there is any value in setting $_SESSIONS['conditions'] as it would either be set to 0 or NULL, neither of which matter as the tick box would be unticked unless $_SESSIONS['conditions'] was set to 1.

still happy to change if you disagree!

Reply:
OK, just looked like it was a removed line.